analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -73,6 +73,11 @@ private const val ROOT_DEPENDENCIES_SCRIPT = "scripts/bundler_root_dependencies.
  */
 private const val RESOLVE_DEPENDENCIES_SCRIPT = "scripts/bundler_resolve_dependencies.rb"
 
+/**
```

Comment:
Commit-message: Would it be worth mentioning what issue the previous implementation had which the new hasn't anymore ?

Reply:
I was hoping that's already implied by the current commit message:

> Do not hard-code "bundler"

The previous implementation hard-code (only) the "bundler" implementation, which obviously does not work anymore if the dependencies required by the helper scripts ever change.

> consider all project scopes, not just the runtime scope.

So previously, only the "runtime" scope was considered.

What information exactly are you missing from here?

Reply:
> @itrich this should fix your issue.

I was wondering what was mentioned here.

Reply:
See https://ort-talk.slack.com/archives/C9NNJ54B1/p1658731298911489?thread_ts=1658512604.176269&cid=C9NNJ54B1.